src/resources/js/tickets.js

```diff
@@ -424,6 +426,26 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 			} );
 		}
 
+		/**
+		 * If the user attempts to nav away without saving global stock setting
+		 * changes then try to bring this to their attention!
+		 */
+		var beforeUnload = function( event ) {
+			var returnValue = false;
+
+			// If we are not on the base panel we alert the user about leaving
+			if ( $metabox.find( '.ticket_panel' ).filter( '[aria-hidden="false"]' ).is( $base_panel ) ) {
```

Comment:
Would this be more performant (no DOM hunting) if we just did (rough code)
```
if ( false === $base_panel.attr( 'aria-hidden' ) ) {
```

Reply:
So the idea here is that if in the future the `$base_panel` is replaced it would still trigger the message.